tests/unit_tests/test_wave_06.py

```diff
@@ -228,7 +257,12 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result is False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_c]
+    assert jesse.inventory == [item_d, item_e, item_f]
```

Comment:
Be careful with `==` asserts for lists! This requires that the items in the list are in the exact same order for it to be true. It would be safer to use `in` like in the previous tests.